.github/workflows/foundry.yml

```diff
@@ -30,5 +30,6 @@ jobs:
       - name: Run Forge build
         run: |
           forge --version
+          forge soldeer update
```

Comment:
@naps62 fyi since I added this after you approved. CI was failing with just the remappings change so needed to also update dependencies here